src/Gui/GLPainter.h

```diff
@@ -24,24 +24,18 @@
 #ifndef GUI_GLPAINTER_H
 #define GUI_GLPAINTER_H
 
-#ifdef FC_OS_WIN32
-#ifndef NOMINMAX
-#define NOMINMAX
-#endif
-#include <windows.h>
-#endif
-#ifdef FC_OS_MACOSX
-#include <OpenGL/gl.h>
-#else
-#include <GL/gl.h>
-#endif
-
 #include <Base/BaseClass.h>
 #include <FCGlobal.h>
-#include <QtOpenGL.h>
 #include <QPoint>
 
 class QPaintDevice;
+class QOpenGLWidget;
+
+# ifdef FC_OS_MACOSX
+# include <OpenGL/gl.h>
+# else
+# include <GL/gl.h>
+# endif
```

Comment:
Something is funny with the indentation here, copy paste maybe?
```suggestion
#ifdef FC_OS_MACOSX
#include <OpenGL/gl.h>
#else
#include <GL/gl.h>
#endif
```